### ISSUE TITLE
External scripts are stripped out when bundling

### ISSUE THREAD

**Issue by web-padawan** [Polymer/polymer-bundler#436]
When using `google-chart` element with the `polymer-build` 0.8.1 and `polymer-bundler` 2.0.0-pre.6, [this line](https://github.com/GoogleWebComponents/google-chart/blob/master/charts-loader.html#L4) is stripped out of the bundle, which results in the following error:
```
Uncaught ReferenceError: google is not defined
    at HTMLElement.<anonymous> (sandbox-demo-first-sub-page.js:1)
    at i.complete (sandbox-app.js:2)
    at boundComplete (sandbox-app.js:2)
```

I updated my [sandbox](https://github.com/web-padawan/polymer-sandbox) to reproduce this issue, see it live [here](https://sandbox-e0d4b.firebaseapp.com/demo/first/) and open console to see that error.

AFAIK there used to be a `NoopResolver` for such scripts when using `vulcanize` and `hydrolysis`. Looks like this logic is broken for some reasons.

**Comment by justinfagnani** [comment]
Fixed by #487

**Comment by web-padawan** [comment]
@justinfagnani should this be fixed by `polymer-build` 0.8.2? It still doesn't work for my sandbox.

The only code I get in bundled build is this one: 
```
        <dom-module id="google-chart" assetpath="../../../vendor/google-chart/">
            <template>
                <style include="google-chart-styles"></style>
                <google-chart-loader id="loader" type="[[type]]"></google-chart-loader>
                <div id="chartdiv"></div>
            </template>
        </dom-module>
```

No external script tag.

**Comment by usergenic** [comment]
This isn't actually fixed until #439 is merged in.

**Comment by usergenic** [comment]
(And then of course Polymer/polymer-build will need to use new release of Polymer/polymer-bundler with that.)
